### PR TITLE
update zulu base image

### PR DESCRIPTION
**Pull request by Ewocker**

### CODE REVIEW

File: zulu-11/Dockerfile [hunk 1]
```diff
@@ -4,13 +4,9 @@ FROM gcc:9.2.0 as dumb-init-builder
...
 RUN cd dumb-init && make
 
-FROM docker.io/azul/zulu-openjdk-debian:11.0.7-11.39.15
+FROM docker.io/azul/zulu-openjdk-debian:11.0.8-11.41.23
 COPY --from=dumb-init-builder /dumb-init/dumb-init /usr/local/bin/dumb-init
 
-RUN apt-get update && \
-   DEBIAN_FRONTEND=noninteractive apt-get install --assume-yes \
-       # These packages are security updates
-       libgnutls30=3.6.7-4+deb10u4 \
```
Comment (nr17) [inline zulu-11/Dockerfile:13]: Did you not just add these libs in a previous PR? why are we getting rid of it now?

File: zulu-8/Dockerfile [hunk 2]
```diff
@@ -4,12 +4,9 @@ FROM gcc:9.2.0 as dumb-init-builder
 RUN git clone -b v1.2.2 --depth 1 --single-branch https://github.com/proofpoint/dumb-init.git
 RUN cd dumb-init && make
 
-FROM docker.io/azul/zulu-openjdk-debian:8u252-8.46.0.19
+FROM docker.io/azul/zulu-openjdk-debian:8u265-8.48.0.53
 
-RUN apt-get update && \
-   DEBIAN_FRONTEND=noninteractive apt-get install --assume-yes \
-       # These packages are security updates
-       libsystemd0=241-7~deb10u4
```
Comment (nr17) [inline zulu-8/Dockerfile:12]: is this intentional?

File: zulu-11/Dockerfile [hunk 3]
```diff
@@ -4,13 +4,9 @@ FROM gcc:9.2.0 as dumb-init-builder
...
-       libsystemd0=241-7~deb10u4
+RUN apt-get update
```
Comment (johngmyers) [inline zulu-11/Dockerfile]: Why run apt-get update at this point?

Before we'd comment out the lines to make subsequent updates easier. See f851d8f4ff5d6363de26ea29c757ae5ced4c3fd7

File: zulu-11/Dockerfile (hunk 1 above)
Comment (johngmyers) [reply zulu-11/Dockerfile:13]: Because the new Zulu base image presumably already incorporates these.

**Review by johngmyers — APPROVED** [review]

**Pull request merged**